site/docs/v1/tech/guides/advanced-registration-forms.adoc

```diff
@@ -0,0 +1,540 @@
+---
+layout: doc
+title: Advanced Registration Forms
+description: Advanced Registration Forms
+---
+:page-liquid:
+
+== What Are Advanced Registration Forms
+
+Advanced registration forms let you build powerful, multi-step, custom registration experiences with no coding required.
+
+image::guides/advanced-registration-forms/built-out-advanced-form.png[Example of built out advanced registration form.,width=1200,role=box-shadow margin-right]
+
+You might be interested in this feature if you use the FusionAuth themed login pages for your application, and the default self service registration form doesn't meet your needs. Whether you want to break a form up into multiple steps for a better user experience, gather user consents, or have the user provide app specific data, advanced registration forms can help.
+
+If you are building your own login and registration pages using the link:../apis[APIs], you can still use the form builder in the administrative user interface, but you will have to generate the user facing HTML from the configured form data and recreate any front end logic. 
```

Comment:
This is true I suppose, but all of the business logic is in front end services. There would be no value in using these on your own because you'd still have to validate the input, perform type conversion, etc. 

Perhaps we should instead indicate, that if you are building your own registration form, you should consider using ours.

Reply:
I modified this to include that message, but I just want folks to understand that if they are building against the APIs, this feature is of minimal use.